indexers/keyvalue/PostgreSQLDBIndexer/README.md

```diff
@@ -0,0 +1,70 @@
+# PostgressIndexer
+
+Indexer wrapper around Postgress database. Postrges is an open source object-relational database. You can read more about it here: https://www.postgresql.org/
+
+
+## Snippets:
+
+Initialise PostgresIndexer:
```

Comment:
Use the same name everywhere as the class, which should be `PostgreSQLDBMSIndexer`